src/modules/services/services.conf

```diff
@@ -1,20 +1,24 @@
 ---
```

Comment:
Should get a top-of-file comment "systemd services configuration. this allows post-install changes to systemd configuration by *something-something-something*".